README.md

```diff
@@ -1,3 +1,244 @@
 # React Breakpoints Mixin
 
 BreakpointsMixin for your React Components.
+
+
+## Example
+
+See the working [example page](http://ui.deseretdigital.com/react-breakpoints-mixin/).
+
+
+## Install
+
+The package is available through npm and also bower.
+
+### npm
+
+```shell
+npm install --save react-breakpoints-mixin
+```
+
+
+### bower
+
+```shell
+bower install --save react-breakpoints-mixin
+```
+
+
+## Import
+
+The `BreakpointsMixin` module is packaged in the [UMD][GoogleUMD] format, which
+means you can bring it in as a [CommonJS][GoogleCommonJS] module using `require`,
+or as a [global variable][GoogleGlobalVariable] using a `<script>` tag.
```

Comment:
I just don't have experience with AMD syntax and didn't want to take the time to figure it out and verify it right now and decided to only cover our common use cases.